### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jlbilh**
Minimal unit tests and initial Github Actions workflow

### CODE REVIEW

File: .github/workflows/github-test.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+run-name: ${{ github.actor }} has invoked Github Test Action
+on:
+  push:
+    branches: ["main", "develop"]
+  pull_request:
+    branches: ["main", "develop"]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (rhowes1) [inline .github/workflows/github-test.yml:11]: This code review process is excellent.

File: js/isItFriday.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function isItFriday(date) {
+    let day = date.getDay();
+    return (day === 5);
```
Comment (rhowes1) [inline js/isItFriday.js]: Can we avoid hard coded values in-line? Change the 5 to a variable?

**Review by rhowes1 — CHANGES_REQUESTED** [review]
Requesting Changes

File: js/isItFriday.js (hunk 2 above)
Comment (jlbilh) [reply js/isItFriday.js]: Added day-of-the-week variables to the latest push!

**Review by jlbilh — COMMENTED** [review]
Reviewed.

**Review by rhowes1 — APPROVED** [review]

**Pull request merged**